### ISSUE TITLE
Default View Encapsulation Behavior

### ISSUE THREAD

**Issue by Karasuni** [angular/angular-cli#8533]
### Versions
```
Angular CLI: 1.5.2
Node: 8.9.1
OS: win32 x64
Angular: 5.0.2
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.0.0-rc0
@angular/cli: 1.5.2
@angular/material: 5.0.0-rc0
@angular-devkit/build-optimizer: 0.0.32
@angular-devkit/core: 0.0.20
@angular-devkit/schematics: 0.0.35
@ngtools/json-schema: 1.1.0
@schematics/angular: 0.1.2
typescript: 2.6.1
webpack: 3.8.1
```

### Repro steps
* Initialize a new project: `ng new my-app`
* `cd my-app`
* Initialize a new component: `ng g c test`

### Observed behavior
The new component will now have `encapsulation: ViewEncapsulation.None` in it's `@Component` decorator. There is no specification of the encapsulation mode in either angular-cli.json or the generate component command.

```
@Component({
  selector: 'app-test',
  templateUrl: './test.component.html',
  styleUrls: ['./test.component.scss'],
  encapsulation: ViewEncapsulation.None
})
```

### Desired behavior
1. In prior versions the `encapsulation` was not defined.
2. The [CLI schema documentation](https://github.com/angular/angular-cli/wiki/angular-cli) does not declare the default encapsulation mode.
3. The [Generate Component documentation](https://github.com/angular/angular-cli/wiki/generate-component) does not specify the default encapsulation mode.
4. Angular's [default behavior](https://angular.io/api/core/ViewEncapsulation) would be to have no specification of the View Encapsulation mode, defaulting to `Emulated` mode.

**Comment by hansl** [comment]
It seems that your `@schematics/angular` package is a bit behind. Could you try to upgrade to 0.1.5 (the latest version) and see if that corrects it?  We had some fixes added to the component especially for this bug.

**Comment by Karasuni** [comment]
@hansl I don't know where `@schematics/angular` is coming from. It's not in the `package.json` that is generated by `@angular/cli 1.5.2` and my cli is the latest version. I've checked outdated packages in my global scope and my project scope with `ncu` and everything is up to date.

**Comment by Karasuni** [comment]
I see https://github.com/angular/devkit/commit/d7c99fac973e4132e1bfca824f14c886b4c06c4a is a possible fix to this issue. The diff doesn't tell me where `ViewEncapsulation.None` comes from but does specify that the expected behaviour should not set the encapsulation.

I don't understand how `@schematics/angular` (or [angular/devkit](https://github.com/angular/devkit1)) is related to the cli. Does this mean that `@angular/cli v1.5.2` comes bundled with `@schematics/angular v0.1.2`?

Once we figure this one out I might update the wiki to include the default settings.

**Comment by hansl** [comment]
Could you clean your `node_modules/` and re-run `npm install`? It's a dependency of the `@angular/cli`.

**Comment by Karasuni** [comment]
Complete reinstall of the global `@angular/cli` and removing and reinstalling the application's `node/modules` still resolves the same dependencies:

```
Angular CLI: 1.5.2
Node: 8.9.1
OS: linux x64
Angular: 5.0.2
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.0.0-rc0
@angular/cli: 1.5.2
@angular/material: 5.0.0-rc0
@angular-devkit/build-optimizer: 0.0.32
@angular-devkit/core: 0.0.20
@angular-devkit/schematics: 0.0.35
@ngtools/json-schema: 1.1.0
@schematics/angular: 0.1.2
typescript: 2.6.1
webpack: 3.8.1
```

(Note that this instance in being run on Bash on Ubuntu on Windows, the other instance from OP is Windows 7. Both instances installed the same project and the same global and local versions.)

**Comment by Karasuni** [comment]
I notice my project's `package-lock.json` specifically requires `0.1.2`. Perhaps there is a bug in npm where it does not update requirement versions properly.

```
"requires": {
  "@schematics/angular": "0.1.2",
}
```

In [`@angular/cli`](https://github.com/angular/angular-cli/tree/v1.5.2) we can find the following:

```
// package.json
"dependencies": {
  "@schematics/angular": "~0.1.0",
}

// package.lock.json
"@schematics/angular": {
  "version": "0.1.0",
}
```

Now I have cli installed globally to allow `ng` to be used in the path. The cli is also installed locally as it's a dependence of the project. If the cli is installed in the project's `node_modules` will it use these packages instead?

**Comment by Karasuni** [comment]
Confirmed removing `package-lock.json` correctly resolved the `0.1.5` dependency and generated a component without encapsulation. Thank you for your help @hansl.

I'll personally dig a bit further in trying to see why `package-lock.json` did update the `@angular/cli` version but not that of it's dependencies.
